doc/manual/manual.tex

```diff
@@ -9937,6 +9937,29 @@ \subsubsection{Benchmarks for operator splitting}
 
 The results shown here can be reproduced using the bash scripts \texttt{run.sh} in the corresponding benchmark folders. 
 
+\subsubsection{3D thermal convection benchmark}
+ilabel{sec:benchmark-thermal-convection}
+
+\textit{This section was contributed by L. Jeanniot and C. Thieulot.}
+
+We use an incompressible and isoviscous Rayleigh-Bénard thermal convection benchmark at infinite Prandtl number in a spherical shell to compare the results predicted by ASPECT with the previously benchmarked numerical code CitcomS. This benchmark follows the setup of Arrial et al. (2014) and Zhong et al. (2008). We explore the sensitivity of the initial and steady-state root mean square velocity $V_RMS$, average temperature $<T>$ and Nusselt number $Nu$ measured at both inner and outer boundaries to resolution for a Rayleigh number $Ra = 7000$ and $Ra = 70000$. 
+
+The setup is a 3d spherical shell with a nondimensional length scale chosen as the approximate thickness of the mantle of the Earth (i.e. mantle thickness = 1). For low resolution models (here global refinement of 3), specifying Cells along circumferences at 12 allows better and quickier convergence to the solution. Angular momentum is here removed from the velocity field at the end of the Stokes solver step so that the net angular momentum is zero because for some geometries and certain boundary conditions the velocity field in not uniquely determined but contains free translations and/or rotations. 
+The dynamics of the fluid are entirely governed by the Rayleigh number $Ra$, which can be interpreted as a ratio of the destabilizing force due to the buoyancy of the heated fluid to the stabilizing force due to the the viscosity of the fluid and heat transfer by conduction. Therefore, we set here all the parameters which are parts of the Rayleigh formula to 1, except for the gravity component. The value given to the gravity magnitude therefore specifies the Rayleigh number value.
```

Comment:
`Therefore, we set here all the parameters which are parts of the Rayleigh formula to 1, except for the gravity component.` We do not usually call it `Rayleigh formula`. I would reformulate this to: `Therefore, we set all the parameters which are part of the Rayleigh number to 1, except for the gravity, which is then identical to the Rayleigh number.`